doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -254,7 +254,7 @@ to the specified type. For example, the following call:
 ccall((:foo, "libfoo"), Cvoid, (Int32, Float64), x, y)
 ```
 
-will behave as if the following were written:
+will behave as if it were written as:
```

Comment:
"written as" -> "written"

Reply:
Could just drop final 'as'. I can't seem to make any more changes after the first I made above.

Reply:
Done